docs/resources/scenarios/noaa-data-fetch.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 'Office Scripts sample scenario: Graph water-level data from NOAA'
+description: 'A sample that fetches JSON data from a NOAA database and uses it to create a chart.'
+ms.date: 12/28/2020
+localization_priority: Normal
+---
+
+# Office Scripts sample scenario: Graph water-level data from NOAA
+
+In this scenario, you need to plot the water level at the [National Oceanic and Atmospheric Administration's Seattle station](https://tidesandcurrents.noaa.gov/stationhome.html?id=9447130). You'll use external data to populate a spreadsheet and create a chart.
+
+You'll develop a script that uses the `fetch` command to query the [NOAA Tides and Currents database](https://tidesandcurrents.noaa.gov/). That will get the water level recorded across a given time span. The information will be returned as JSON, so part of the script will translate that into range values. Once the data is in the spreadsheet, it will be used to make a chart.
+
+## Scripting skills covered
+
+- External API calls (`fetch`)
+- JSON parsing
+- Charts
+
+## Setup instructions
+
+1. Open the workbook with Excel for the web.
+
+1. Under the **Automate** tab, open **All Scripts**.
+
+1. In the **Code Editor** task pane, press **New Script** and paste the following script into the editor.
```

Comment:
Also a button. Another reason not to use 'press' is that it is less inclusive -- it is mouse-centric and implies one is not using keyboard or voice navigation. 
```suggestion
1. In the **Code Editor** task pane, select **New Script** and paste the following script into the editor.
```